metaflow/metaflow_config.py

```diff
@@ -68,6 +102,31 @@ def from_conf(name, default=None):
     if from_conf("METAFLOW_DATASTORE_SYSROOT_S3")
     else None,
 )
+
+DATATOOLS_DEFAULT_CLIENT_PARAMS = from_conf(
+    "METAFLOW_DATATOOLS_CLIENT_PARAMS",
+    '{"endpoint_url": %s, "verify": %s}' % (S3_ENDPOINT_URL, S3_VERIFY_CERTIFICATE)
+    if S3_ENDPOINT_URL and S3_VERIFY_CERTIFICATE
+    else "{}",
+)
+DATATOOLS_DEFAULT_SESSION_VARS = from_conf("METAFLOW_DATATOOLS_SESSION_VARS", "{}")
```

Comment:
nit^2: maybe move this line closer to where the value is parsed at line 122. At the first glance I thought it was a third configuration variable.